/**:
  ros__parameters:
    # boundary inflation (meters)
    region_inflation: 0.075

    # Collection =
    collection_area: [
      0.1, 0.25, 1.1, 1.3,
      1.05, 1.25, 0.725, 0.875,
      1.75, 1.95, 0.725, 0.875,
      2.75, 2.9, 1.1, 1.3,
      0.1, 0.25, 0.3, 0.5,
      1.0, 1.2, 0.1, 0.25,
      2.75, 2.9, 0.3, 0.5   
    ]

    # Pantry 
    pantry_area: [
      1.15, 1.35, 1.35, 1.55,
      1.05, 1.85, 1.35, 1.55,
      0.0, 0.2, 0.7, 0.9,
      0.7, 0.9, 0.7, 0.9,
      1.4, 1.6, 0.7, 0.9,
      2.1, 2.3, 0.7, 0.9,
      2.8, 3.0, 0.7, 0.9,
      0.6, 0.8, 0.0, 0.2,
      1.4, 1.6, 0.0, 0.2,
      2.2, 2.4, 0.0, 0.2
    ]